### PR TITLE
SUS-2535 | Caching for image review history requests

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/SUS-2535

### CODE REVIEW

File: includes/wikia/tasks/Tasks/ImageReviewTask.class.php [hunk 1]
```diff
@@ -119,8 +120,9 @@ public function delete( $pageList, $suppress = false ) {
 	public function update( $pageList ) {
 		foreach ( $pageList as list( $cityId, $pageId, $revisionId ) ) {
 			$db = \WikiFactory::IDtoDB( $cityId );
-			$key = wfForeignMemcKey( $db, '', "image-review-${cityId}-${pageId}-${revisionId}" );
-			\WikiaDataAccess::cachePurge( $key );
+			$key = wfForeignMemcKey( $db, 'image-review', $cityId, $pageId, $revisionId );
```
Comment (macbre) [inline includes/wikia/tasks/Tasks/ImageReviewTask.class.php]: Can we use cityId here to avoid `\WikiFactory::IDtoDB` call?

**Review by macbre — APPROVED** [review]
One minor thing left

File: includes/wikia/tasks/Tasks/ImageReviewTask.class.php (hunk 1 above)
Comment (macbre) [reply includes/wikia/tasks/Tasks/ImageReviewTask.class.php]: Actually, do we need `$db` here? `$cityId` is a part of the memcache key.

**Review by macbre — APPROVED** [review]

File: extensions/wikia/ImageReview/ImageReviewImageStatus.php [hunk 2]
```diff
@@ -33,7 +33,13 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 
 	if ( in_array( $wgCityId, $wgImageReviewTestCommunities ) ) {
 		$reviews =
-			fetchReviewHistoryFromService( $wgCityId, $imagePage->getID(), $imagePage->getTitle()->getLatestRevID() );
+			fetchReviewHistoryFromService( $wgCityId, $imagePage->getTitle() );
+
+		// TODO: temporary solution, remove it as soon as image review history is migrated to the new tool
```
Comment (WikiaTech) [inline extensions/wikia/ImageReview/ImageReviewImageStatus.php:38]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**